src/create_update.c

```diff
@@ -401,73 +530,15 @@ int main(int argc, char **argv)
 		}
 		manifest->includes = manifest_includes;
 	}
-	while (1) {
-		char *group = next_group();
-		struct manifest *oldm;
-		struct manifest *newm;
-
-		if (!group) {
-			break;
-		}
-
-		if (strcmp(group, "os-core") == 0) {
-			continue;
-		}
 
-		printf("Processing bundle %s\n", group);
-
-		/* Step 4: Make a manifest for this functonal group */
-		oldm = g_hash_table_lookup(old_manifests, group);
-		newm = g_hash_table_lookup(new_manifests, group);
-		add_component_hashes_to_manifest(newm, new_full);
-		apply_heuristics(oldm);
-		apply_heuristics(newm);
-		newm->prevversion = oldm->version;
-
-		/* add os-core as an included manifest */
-		if (!manifest_includes(oldm, "os-core")) {
-			oldm->includes = g_list_prepend(oldm->includes, old_core);
-		}
-		if (!manifest_includes(newm, "os-core")) {
-			newm->includes = g_list_prepend(newm->includes, new_core);
-		}
-
-		/* Step 5: Subtract the core files from the manifest */
-		subtract_manifests_frontend(oldm, oldm);
-		subtract_manifests_frontend(newm, newm);
-
-		/* Step 6: Compare manifest to the previous version... */
-		if (match_manifests(oldm, newm) == 0 && !changed_includes(oldm, newm)) {
-			LOG(NULL, "", "%s components have not changed, no new manifest", group);
-			printf("%s components have not changed, no new manifest\n", group);
-			/* Step 6a: if nothing changed, stay at the old version */
-			newm->version = oldm->version;
-		} else {
-			apply_heuristics(newm);
-#warning missing rename_detection here
-			/* Step 6b: otherwise, write out the manifest */
-			old_deleted = remove_old_deleted_files(oldm, newm);
-			sort_manifest_by_version(newm);
-			type_change_detection(newm);
-			newfiles = prune_manifest(newm);
-			if (newfiles > 0 || old_deleted > 0 || changed_includes(oldm, newm)) {
-				LOG(NULL, "", "%s component has changes (%d new, %d deleted), writing out new manifest", group, newfiles, old_deleted);
-				printf("%s component has changes (%d new, %d deleted), writing out new manifest\n", group, newfiles, old_deleted);
-				if (write_manifest(newm) != 0) {
-					LOG(NULL, "", "%s component manifest write failed", group);
-					printf("%s component manifest write failed\n", group);
-					goto exit;
-				}
-				create_manifest_deltas(newm, manifests_last_versions_list);
-			} else {
-				LOG(NULL, "", "%s component has not changed (after pruning), no new manifest", group);
-				printf("%s component has not changed (after pruning), no new manifest\n", group);
-				newm->version = oldm->version;
-			}
-		}
-
-		nest_manifest(new_MoM, newm);
-	}
+	manifests->new_full = new_full;
+	manifests->old_core = old_core;
+	manifests->new_core = new_core;
+	manifests->old_manifests = old_manifests;
+	manifests->new_manifests = new_manifests;
+	manifests->manifests_last_versions_list = manifests_last_versions_list;
```

Comment:
Looks like os-core manifest deltas are still being created. Can that be removed too?